ui/shared/status/StatusEmojiPopup.qml

```diff
@@ -76,18 +80,104 @@ Popup {
+
+            var skinColors = ["1f3fb", "1f3fc", "1f3fd", "1f3fe", "1f3ff"];
+            var emojisWithColors = [
+                        "1f64c",
+                        "1f44f",
+                        "1f44b",
+                        "1f44d",
+                        "1f44e",
+                        "1f44a",
+                        "270a",
+                        "270c",
+                        "1f44c",
+                        "270b",
+                        "1f450",
+                        "1f4aa",
+                        "1f64f",
+                        "261d",
+                        "1f446",
+                        "1f447",
+                        "1f448",
+                        "1f449",
+                        "1f595",
+                        "1f590",
+                        "1f918",
+                        "1f596",
+                        "270d",
+                        "1f485",
+                        "1f442",
+                        "1f443",
+                        "1f476",
+                        "1f466",
+                        "1f467",
+                        "1f468",
+                        "1f469",
+                        "1f471",
+                        "1f474",
+                        "1f475",
+                        "1f472",
+                        "1f473",
+                        "1f46e",
+                        "1f477",
+                        "1f482",
+                        "1f385",
+                        "1f47c",
+                        "1f478",
+                        "1f470",
+                        "1f6b6",
+                        "1f3c3",
+                        "1f483",
+                        "1f647",
+                        "1f481",
+                        "1f645",
+                        "1f646",
+                        "1f64b",
+                        "1f64e",
+                        "1f64d",
+                        "1f487",
+                        "1f486",
+                        "1f6a3",
+                        "1f3ca",
+                        "1f3c4",
+                        "1f6c0",
+                        "26f9",
+                        "1f3cb",
+                        "1f6b4",
+                        "1f6b5",
+                        "1f3c7",
+                        "1f575"
+                    ]
+
+
+            if (skinColor !== "") {
+                if (emoji.unicode.includes(skinColor)) {
+                    newCategories[categoryNames[emoji.category]].push(Object.assign({}, emoji, {filename: emoji.unicode + '.svg'}));
+                } else {
+                    if (!emojisWithColors.includes(emoji.unicode) &&!emoji.unicode.includes("1f3fb") && !emoji.unicode.includes("1f3fc") && !emoji.unicode.includes("1f3fd")
+                            && !emoji.unicode.includes("1f3fe") && !emoji.unicode.includes("1f3ff"))  {
```

Comment:
I think we could make a small function to simplify those conditions
Something like
```qml
function containsSkinColor(code) {
  return skinColors.some(function (color) {
    return code.includes(color)
  });
}
```

Then you can call this function like this:
```qml
if (!emojisWithColors.includes(emoji.unicode) && !containsSkinColor(emoji.unicode))  {
```

Disclaimer: code is untested and was written quickly

Reply:
Tested and seemed working :)